modules/sharedaddy/sharing-sources.php

```diff
@@ -623,20 +623,28 @@ public function get_display( $post ) {
 		$via = $this->sharing_twitter_via( $post );
 
 		if ( $via ) {
-			$via = '&via=' . rawurlencode( $via );
+			$via = 'data-via="' . esc_attr( $via ) . '"';
 		} else {
 			$via = '';
 		}
 
 		$related = $this->get_related_accounts( $post );
 		if ( ! empty( $related ) && $related !== $via ) {
-			$via .= '&related=' . rawurlencode( $related );
+			$related = 'data-related="' . esc_attr( $related ) . '"';
+		} else {
+			$related = '';
 		}
 
 		if ( $this->smart ) {
 			$share_url = $this->get_share_url( $post->ID );
 			$post_title = $this->get_share_title( $post->ID );
-			return '<div class="twitter_button"><iframe allowtransparency="true" frameborder="0" scrolling="no" src="' . esc_url( $this->http() . '://platform.twitter.com/widgets/tweet_button.html?url=' . rawurlencode( $share_url ) . '&count=none&text=' . rawurlencode( $post_title . ':' ) . $via ) . '" style="width:55px; height:20px;"></iframe></div>';
```

Comment:
The selector `.sd-social-official .sd-content .twitter_button iframe` in `modules/sharedaddy/sharing.css` should be removed.